lib/puppet/face/man.rb

```diff
@@ -39,12 +39,28 @@
     EOT
 
     default
-    when_invoked do |name, options|
-      if legacy_applications.include? name then
-        return Puppet::Application[name].help
+    when_invoked do |*args|
+      # [<argument> ...] <options>
```

Comment:
What's up with this comment? Was it to just a reminder of what `*args` will be?

Reply:
Yeah, I was trying to give the next person looking at the code a chance to understand what your getting passed in.

So the these command lines become the appropriate call.
```
"puppet man" -> man([{}])
"puppet man cert" -> man(["cert", {}])
"puppet man man" -> man([{}])    # yes that's right,  thank you face_base
"puppet man cert man" -> man(["cert", "man", {}])
```

Reply:
If that's the intention, we should add a bit more context here so it's clear. Something like `*args will evaluate to...` or whatever. Also, this is currently a bit misleading since `<options>` are outside of the args array. `[<arg1>,... , {<options>}]` would be a more accurate representation.